smart-contract-verifier/smart-contract-verifier-server/tests/solidity_types.rs

```diff
@@ -105,7 +110,8 @@ impl TestCase for Flattened {
             "metadata": {
                 "chainId": self.chain_id,
                 "contractAddress": self.contract_address
-            }
+            },
+            "postActions": ["lookup-methods"]
```

Comment:
I'm not sure about this, but maybe empty postActions in case `expected_lookup_methods.is_none()` ?

Reply:
Fixed